src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Prefabs/Logout.cs

```diff
@@ -10,6 +11,12 @@ public async void OnLogout()
         // Remove the saved "remember me" data, if any
         PlayerPrefs.DeleteKey(Login.PlayerAccountKey);
 
+        // disconnect wallet
+        if (Web3Accessor.Web3.Signer is WebPageWallet wallet)
```

Comment:
Duplicating scripts in the repo?

Reply:
yeah we're duplicating sample folder changes using a GitHub Action, instead of doing it manually it happens on a PR to main

Reply:
can we gitignore the copy somehow?

Reply:
no, depending on which sample we ignore (imported/importable) we'll either need to duplicate manually which defeats the whole process of automation or we won't have an importable sample in the package, we discussed this at length with @kalambet and this was the best solution

Reply:
can we keep just the ones in the Package/Sample?

Reply:
Package/Sample~ is ignored in Unity so any changed to sample would need to be made to the imported sample and duplicate those changes to package/sample~, we can unignore package/sample~ in Unity but that would mean we need to make sample changes in the package environment not project environment which could lead to unpredictable issues, plus it's against common practice, which is to ignore sample folder in Unity https://docs.unity3d.com/Manual/cus-samples.html

Reply:
Let's talk about it on the next tech grooming session.